docs/INSTALL.rst

```diff
@@ -199,7 +199,50 @@ contains ``haas.cfg``. This pre-allocation is easier to reason about
 than on-demand creation, and allows HaaS to be run as an unprivileged user,
 but it also causes some limitations.  For instance, because of this, headnodes
 can only be connected to networks with allocated VLANs.  The bridges must also
-be pre-allocated again on each boot. For now, the recomended approach is to add::
+be pre-allocated again on each boot. For now, the recommened method is to use 
+systemd.  A systemd service for running the create_bridges script is available 
+in the 'scripts' directory.
+
+Name of the service is: create_bridges.service
+
+Name of the script is: create_bridges
```

Comment:
Also mark the `create_bridges.service` and `create_bridges`, there are couple of same spots below.